### PR TITLE
Feature/products endpoint

### PR DESCRIPTION
**Pull request by 132dannys**

### CODE REVIEW

File: api/anri/apps/products/models/tag.py
Comment (ivanboika) [inline api/anri/apps/products/models/tag.py:1]: Should I delete Tag class from my pr?

Comment (132dannys) [reply api/anri/apps/products/models/tag.py:1]: Yes, please

File: api/poetry.lock
Comment (Orange-hanter) [inline api/poetry.lock:1]: gitignor?

**Review by ivanboika — APPROVED** [review]

File: api/anri/apps/products/models/product.py [hunk 1]
```diff
@@ -8,5 +8,7 @@ class Product(CoreModel):
     code = models.CharField(max_length=127)
     group = models.CharField(max_length=127)
     description = models.CharField(max_length=127)
```
Comment (Orange-hanter) [inline api/anri/apps/products/models/product.py:10]: Description could be longer. Increase it to 512 characters or use dynamic length type

**Review by Orange-hanter — APPROVED** [review]

File: api/poetry.lock
Comment (132dannys) [reply api/poetry.lock:1]: done

**Pull request merged**